frontend/src/lib/lemon-ui/Tooltip/Tooltip.tsx

```diff
@@ -109,7 +109,7 @@ export function Tooltip({
     )
 
     if (!title) {
-        return <>{children}</>
+        return <>{child}</>
```

Comment:
fly by: just above we check if `children` is invalid
creating `child` in the process

let's not use children after we know it might be invalid